client/blocks/product-purchase-features-list/mobile-apps.jsx

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+
+export default localize( ( { translate } ) => {
+	return (
+		<div className="product-purchase-features-list__item">
+			<PurchaseDetail
+				icon={ <img alt="" src="/calypso/images/illustrations/jetpack-apps.svg" /> }
+				title={ translate( 'Download our mobile apps' ) }
+				description={ translate(
+					'Manage all your sites from a single dashboard: publish content, ' +
+						'track stats, moderate comments, and more.'
+				) }
+				buttonText={ translate( 'Get the apps' ) }
+				href={
+					'https://apps.wordpress.com/get?utm_source=calypsomyplan&utm_medium=cta&utm_campaign=calypsogetappscard'
+				}
```

Comment:
For string literal JSX props, let's double quotes instead of curly braces plus single quotes, i.e.

```jsx
href="https://apps.wordpress.com/get?utm_source=calypsomyplan&utm_medium=cta&utm_campaign=calypsogetappscard"
```

Also, it looks a bit weird to have the whole campaign/Tracks stuff in the URL. I'm not enough of a Tracks expert to be able to tell if that's the way we're usually doing things. Alternatives that come to mind would include firing a Calypso Tracks event, or check the HTTP referrer on the apps.wordpress.com side and act accordingly there...

Reply:
Thanks a lot @ockham! Agreed about the props 👍 

Re tracking, Calypso side tracking events are not included atm in any of the feature cards. Might be worth changing it later as a general effort towards usage analysis.

Reply:
Seems we have an in-house system called SEM for passing marketing info in the URL, so I believe this is the correct way to do it. apps.wordpress.com just pulls in those SEM funcs.

Using `addQueryArgs()` would make the href more readable.